config/jobs/kubernetes-sigs/cluster-api/cluster-api-periodics-release-1-6.yaml

```diff
@@ -0,0 +1,206 @@
+periodics:
+- name: periodic-cluster-api-test-release-1-6
+  interval: 4h
```

Comment:
Is there a reason we need to use longer interval (4h instead of 2h) throughout the file then what we have on [main](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes-sigs/cluster-api/cluster-api-periodics-main.yaml#L3)?

cc @sbueringer

Reply:
I copied it from v1.5, in the release docs it says to copy from v1.3 which is also 4h.

Reply:
The reason is that we don't want to waste resources on branches which are relatively stable. release-1.6 won't change that much so we don't need a test every 2 hours